### ISSUE TITLE
Allow modification of the 'backgroundColor' property of errorMessageView

### ISSUE THREAD

**Issue by aurora14** [rezzza/VLRTextField#2]
I'm able to customize every property for a text field and related views except for this one. In order to get the behaviour I wanted I had to modify this property within the framework itself, because the 'errorMessageView' label gets created in 'addErrorViewFromError' and thus remains within that scope. 

If I missed something obvious, my apologies - maybe you could point me in the right direction? :) This is a fantastic library otherwise

**Comment by ipodishima** [comment]
Hey! Yes, the error view takes the color from self or from the superview

https://github.com/rezzza/VLRTextField/blob/e11a70f99d7e90b2f9f3969a631a411352ee1c34/Files/VLRTextField.m#L176

This is not working for your case?

**Comment by aurora14** [comment]
Hey! Thanks for the reply :)

After your comment, I decided to take a look at my code again. I solved the issue, but the following may help others who might be stuck on the same problem. 

- If the superview has a 'clear' colour, the background of the error view will be black, so this has to be explicitly set to the desired value. I had a textfield embedded in a table view cell, with the content view having a white background and the textfield set to 'clear' background; thus when the error view tried to look at this property, it behaved accordingly. The solution was simply to set the text field to 'white' in this instance. 

This works well, but it may still be worth considering allowing to set this property explicitly in some future revision

- Original version
![img_5683](https://cloud.githubusercontent.com/assets/16133924/23776995/d1d2d954-0585-11e7-973f-56bbd4d8bdf2.PNG)
 
- Fixed version
![img_5684](https://cloud.githubusercontent.com/assets/16133924/23777035/0903bb28-0586-11e7-8b08-a201d97cc41b.PNG)

Thanks for looking into the issue and quick response :)
